File: sangjin/level1/25_수박수박수박수박수박수.cpp
#include <string>
#include <vector>

using namespace std;

string solution(int n) {
    string answer = "";

    for(int i = 0; i < n/2; i++){
        answer += "수박";
    }

    n%2 == 0 ? answer : answer += "수";
    
    return answer;
}

File: yunj/level1/25_수박수박수박수박수박수.java
class Solution {
    public String solution(int n) {
        String repeat = "수박";
        String odd = "수";
        StringBuffer sb = new StringBuffer();

        int r = n / 2;
        int o = n % 2;

        for (int i= 0; i< r; i++){
            sb.append(repeat);
        }
        if (o == 1){
            sb.append(odd);
        }
        return sb.toString();
    
    }
}